README.md

```diff
@@ -8,24 +8,35 @@
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=org.schemaspy%3Aschemaspy&metric=coverage)](https://sonarcloud.io/dashboard?id=org.schemaspy%3Aschemaspy)
 [![Gitter](https://badges.gitter.im/schemaspy/schemaspy.svg)](https://gitter.im/schemaspy/schemaspy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-**SchemaSpy** is a database metadata analyzer. It helps your database administors and developers visualize, navigate and understand your data model. With an easy-to-use HTML-based report, traversing the entity-relationship diagram has never been simpler. Our product showcase is available at http://schemaspy.org/sample/index.html.
+**SchemaSpy** is a database metadata analyzer. It helps your database administrators and developers visualize, navigate and understand your data model. With an easy-to-use HTML-based report, traversing the entity-relationship diagram has never been simpler. Our product showcase is available at http://schemaspy.org/sample/index.html.
 
 [![SchemaSpy sample](http://schemaspy.org/img/example_page_epivirusurf.png)](http://schemaspy.org/samples/epivirusurf)
 
 # Installation
 
-SchemaSpy is a stand alone tool. Just download the lastest
-[JAR file](https://github.com/schemaspy/schemaspy/releases/download/v6.2.4/schemaspy-6.2.4.jar)
-or [Docker image](https://hub.docker.com/r/schemaspy/schemaspy/) and you're
-ready to go!
+SchemaSpy is a standalone application without GUI. 
+Just download the latest [JAR file](#jar-files-and-dependencies)(see below) or [Docker image](https://hub.docker.com/r/schemaspy/schemaspy/) and you're ready to go!
```

Comment:
I would want 
`Just download the latest [JAR file](https://github.com/schemaspy/schemaspy/releases/atest) or [Docker image]`

The link doesn't work since the header changed. But for non-java people we want to make it as simple as possible and just link to release page.